docs/api/api_configuration.md

```diff
@@ -45,6 +45,70 @@ public class LightManager : NetDaemonRxApp
 
 ```
 
+### Instantiating Strongly Typed Entities
+
+Many of the basic entity types can be created from a string in the YAML. This avoids having to create them later and you have the simple methods like TurnOn/TurnOff ready to go.
+
+**humiditycontrol.yaml**
+
+```yaml
+
+Humidity_control_kitchen:
+    class: HumidityManager
+    humidity_sensor: sensor.kitchen_humidity
+    humidifier: switch.humidifier
+
+```
+
+```csharp
+
+public class HumidityManager : NetDaemonRxApp
+{
+    #region -- Config properties --
+
+    // This property will be automapped with ´the_light´ config
+    public SensorEntity? HumiditySensor { get; set; }
+    public SwitchEntity? Humidifier { get; set; }
+
+    #endregion
+
+    /// <summary>
+    ///     Initialize, is automatically run by the daemon
+    /// </summary>
+    public override void Initialize()
+    {
+        this.RunEvery(TimeSpan.FromMinutes(5), () =>
+            {
+         if (humiditySensor?.State?.GetType() == typeof(System.Int64))
+                {
+                    Int64 humidity = humiditySensor.State;
+                    if (humidity > 80)
+                    {
+                        if (humidifier.State ?? "Unknown" == "off")
+                        {
+                            humidifier.TurnOn();
+                        }
+                    }
+                    else
+                    {
+                        if (humidifier?.State ?? "Unknown" == "on")
+                        {
+                            humidifier.TurnOff();
+                        }
```

Comment:
on line 87 you are using null coalesce operator, and again on 94, but on 94 you are also using the null check operator before state. I guess using either method will work because a null will never be equal to off or on very similar to your unknown value